ducktape/services/service.py

```diff
@@ -51,11 +52,17 @@ class Service(TemplateRenderer):
     # }
     logs = {}
 
-    def __init__(self, context, num_nodes, *args, **kwargs):
+    # TODO: I changed this from num_nodes to nodes but I may not have updated all places where this constructor is called.
```

Comment:
Oh, and now I see that we do the conversion below. I've lost track of previous discussions around this since they happened awhile ago. I guess maybe the compatibility won't be a real issue if we never invoke this constructor with that parameter as a keyword argument.